examples/wifi-echo/server/esp32/main/DeviceCallbacks.cpp

```diff
@@ -45,6 +47,8 @@ using namespace ::chip::DeviceLayer;
 extern LEDWidget statusLED1;
 extern LEDWidget statusLED2;
 extern WiFiWidget wifiLED;
+extern const chip::NodeId kLocalNodeId;
```

Comment:
cold we have headers for these instead of direct extern declarations?

Reply:
should probably be in coding guidelines...